playground/src/components/entry-point/entry-point.jsx

```diff
@@ -20,13 +23,23 @@ setupGlobalErrorListener();
 const apolloClient = configureApolloClient();
 
 const EntryPoint = () => (
-  <ApplicationShell
-    environment={window.app}
-    applicationMessages={loadMessages}
-    apolloClient={apolloClient}
-  >
-    <AsyncPlaygroundRoutes />
-  </ApplicationShell>
+  <Router history={history}>
```

Comment:
A change to allow loading the Custom View iFrame from the same application.

Reply:
Not sure I follow 🤔

Reply:
We are loading both the Playground Custom Application and the Custom View from the same server which renders this same entry point.
When loading the Custom View, we don't wan't to render the `ApplicationShell` (Custom Application) but something totally different (the Custom View demo).